src/silly_sprites/opengl/input.cpp

```diff
@@ -0,0 +1,114 @@
+#include "input.h"
+
+namespace sly::gl {
+    void Input::update_keys(Window const& window) {
+    
+        std::swap(current_keys, last_keys);
+        current_keys.fill(false);
+    
+        for (int i = 0; i < current_keys.size(); ++i) {
```

Comment:
Should be `std::size_t` instead of `int` (or `usize`, according to the styleguide, but there is no header with those type definitions yet, I will open an issue for that).

Reply:
we need to keep this in mind an change it when the header exists.